Add web accessibility details (#6046)
flutter/website: src/docs/development/accessibility-and-localization/accessibility.md

    1. In the Accessibility Inspector,
       select **Audit** in the toolbar, and then
       select **Run Audit** to get a report of potential issues
 
  * For web:
    1. Turn on Screen Reader for your OS
    2. Navigate to the Enable accessibility toggle using the Screen Reader to build the semantics tree. 

>     You can skip this step if you programmatically auto-enable accessibility for your app using this API: `RendererBinding.instance.setSemanticsEnabled(true)`

Reviewer: The instructions above are for developers to inspect the accessibility tree, but the web instructions read like a user guide for accessibility user to use the app. For developers, we could recommend using Chrome DevTools (and equivalents in other browsers) to inspect the HTML tree containing the ARIA attributes generated by Flutter. In Chrome (and probably new Edge) the "Elements" tab has "Accessibility" sub-tab that can be used to inspect the data exported to the semantics tree.

## Screen readers

For mobile, screen readers ([TalkBack][], [VoiceOver][]) enable visually
impaired users to get spoken feedback about the contents of the screen 
and interact with the UI via gestures on mobile and keyboard shortcuts on desktop. 
Turn on VoiceOver or TalkBack on your mobile device and navigate around your app. 

For web, the following screen readers are currently supported:

Mobile Browsers:
* iOS - VoiceOver
* Android - TalkBack

Desktop Browsers:
* MacOS - VoiceOver
* Windows - JAWs & NVDA

> Screen Readers users on web will need to toggle 
> "Enable accessibility" button to build the semantics tree.

Reviewer 2: ```suggestion
On the web, users of screen readers need to toggle 
"Enable accessibility" button to build the semantics tree.
```

## Screen readers

For mobile, screen readers ([TalkBack][], [VoiceOver][]) enable visually

> impaired users to get spoken feedback about the contents of the screen. Turn on VoiceOver or TalkBack on your mobile device and navigate around your app. 

Reviewer: TalkBack and VoiceOver also enable interactions with the UI (via gestures on mobile, and keyboard shortcuts on desktop).

Reviewer 2: Also, please break the line after the period. Thx

For web, the following screen readers are currently supported:

Mobile Browsers:
* iOS - VoiceOver
* Android - TalkBack

Desktop Browsers:
* MacOS - VoiceOver

> * Windows - JAWs

Reviewer: NVDA?

If you run into any issues, use the [`Semantics` widget][] to customize the

> accessibility experience of your app.

Reviewer: nit: I'd phrase this differently, something like: "Flutter's standard widgets generate an accessibility tree automatically. However, if your app needs something different, it can be customized using the Semantics widget".

<iframe width="560" height="315" src="https://www.youtube.com/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

## Testing accessibility on web:

> You can debug accessibility by visualizing the semantic nodes created for your web app using the following command line flag in profile and relese modes:

Reviewer: nit: (applies throughout the file) it seems this file is attempting to keep a certain line length (looks like 80, but I didn't count)

Reviewer 2: Yup! The GitHub rule is to break at 80 and we often break much less than that.

`$ flutter run -d chrome --profile \
 --dart-define=FLUTTER_WEB_DEBUG_SHOW_SEMANTICS=true`
 

> With the flag activated, youll be able to see your semantic nodes on top of the widgets and verify if the semantic elements are placed where they should be. If youfind examples like that please [file a bug report][]. 

Reviewer: s/youll/you'll/
s/youfind/you find/

Reviewer 2: That last sentence was confusing, but is this an accurate change?

```suggestion
With the flag activated, the semantic nodes appear on top of the widgets,
and you can verify that the semantic elements are placed where they should be.
If the semantic nodes are incorrectly placed, please [file a bug report][]. 
```
